app/assess/data.py

```diff
@@ -342,3 +346,34 @@ def get_comments(application_id: str, sub_criteria_id: str):
     )
     comment_response = get_data(comment_endpoint)
     return comment_response
+
+def get_file_url(filename: str, application_id: str):
+    """_summary_: Function is set up to retrieve
+    files from aws bucket.
+    Args:
+        filename: Takes an filename, application_id: Takes an application_id # noqa
+    Returns:
+        Returns a presigned url.
+    """
+    if (Config.FLASK_ENV == "unit_test"):
+        return filename
```

Comment:
Could probably mock/patch the method(s) instead of conditionals, but this is an easy option so not totally against it